delete.php

```diff
@@ -0,0 +1,31 @@
+<?php
+    require_once('functions.php');
+    $db = createDBConnection();
+    if (isset($_POST['yesDelete'])) {
+        $deletedRecipe = deleteRecipe($db);
+        if ($deletedRecipe) {
```

Comment:
could this just be one if with an or?

Reply:
Sadly not but that is because I have realised that it is v wrong so thanks for the heads up 😆